GiEnJul/ClientApp/src/components/institution/InstitutionForm.tsx

```diff
@@ -177,7 +177,7 @@ const RegistrationForm = () => {
             body: JSON.stringify(submit)
         })
         .then((response) => {
-            if (response.status === 201) {
+            if (response.status === 200) {
```

Comment:
Could be changed to 
```suggestion
            if (response.ok) {
```
?

Reply:
dont really need it, but is better for readability